hw/top_earlgrey/data/ip/chip_clkmgr_testplan.hjson

```diff
@@ -144,9 +144,7 @@
               "chip_sw_clkmgr_external_clk_src_for_sw_fast_dev",
               "chip_sw_clkmgr_external_clk_src_for_sw_slow_dev",
               "chip_sw_clkmgr_external_clk_src_for_sw_fast_rma",
-              "chip_sw_clkmgr_external_clk_src_for_sw_slow_rma",
-              "manuf_cp_unlock_raw",
-              "manuf_cp_yield_test"]
```

Comment:
These two are present in `manuf_testplan.hjson`. Are you removing these because they need to be in `chip_sim_cfg.hjson`?

Reply:
Removed because the code in these tests will already be run during provisioning, and the silicon that we will use for sival testing will therefore already have "passed" these tests.

Reply:
Just to close the loop, since you remove bazel target, do you need to set si_stage to 'None' ?

Reply:
I left some other bazel targets, so the testpoint is still tested in sival. I changed si_stage to SV2, since SV1 is unrealistic.